examples/flutter_gallery/ios/Runner/AppDelegate.m

```diff
@@ -0,0 +1,6 @@
+#import <Flutter/Flutter.h>
+#include "AppDelegate.h"
+
+@implementation AppDelegate
+
```

Comment:
any comments / instructions / 'add your stuff here' type messages we should add to the samples and the templates?